projects/wvr-elements/src/lib/wvr-dropdown/wvr-dropdown.component.ts

```diff
@@ -134,6 +392,7 @@ export class WvrDropdownComponent extends WvrAbstractBaseComponent {
   clickOpen($event: Event): void {
     if (this.toggleOn === 'click') {
       this.isOpen() ? this.closeDropdown() : this.openDropdown();
+      $event.stopPropagation();
```

Comment:
Remove this, as it relates to another issue

Reply:
Removed this line.